src/Features/Core/Portable/xlf/FeaturesResources.ru.xlf

```diff
@@ -4200,6 +4200,11 @@ Do you want to continue?</source>
         <target state="translated">метод доступа к событию</target>
         <note />
       </trans-unit>
+      <trans-unit id="required_property">
+        <source>required property</source>
+        <target state="new">Обязательное свойство</target>
```

Comment:
should be the lowercase naming? I was referring to `property_` localization key which starts with the capital letter

Reply:
In general translation should keep the style of the original message. Since original message starts with lowercase, so should the translation. But again, this is a task for a localization team

Reply:
reverted the localizations to English